cmd/snap-update-ns/bootstrap.c

```diff
@@ -371,14 +373,15 @@ static int parse_arg_u(int argc, char * const *argv, int *optind, unsigned long
 	if (uid_out != NULL) {
 		*uid_out = parsed_uid;
 	}
-	*optind += 1; // Account for the argument to -u.
+	*optind += 1;		// Account for the argument to -u.
```

Comment:
Hmmm this looks peculiar, I'd expect a single space.

Reply:
The mysteries of indent.